### DISCUSSION TITLE
https://heywillow.io/ instructions outdated?

### DISCUSSION THREAD

**Discussion by zolstarym** [toverainc/willow-inference-server#115] [General]
At https://heywillow.io/willow-inference-server/#getting-started the instructions seem to be outdated and don't match the githubs'. Can they be updated to match?

**Comment by kristiankielhofner** [comment]
Our top priority was getting a Willow 0.1 release candidate out for existing users to test. When we get enough feedback (things seem to be going very well) and really release 0.1 we'll make sure to have heywillow.io updated for Willow, WAS, and WIS as a central source of documentation.
